logging/cluster-logging-release-notes.adoc

```diff
@@ -1,34 +1,194 @@
 :_content-type: ASSEMBLY
 [id="cluster-logging-release-notes"]
-= Release notes for Red Hat OpenShift Logging 5.2
+= Release notes for Red Hat OpenShift Logging 5.3
 include::_attributes/common-attributes.adoc[]
 :context: cluster-logging-release-notes-v5x
 
 toc::[]
 
-[id="openshift-logging-supported-versions"]
-== Supported versions
-
-.{product-title} version support for Red Hat OpenShift Logging (RHOL)
-[options="header"]
-|====
-|        |4.7          |4.8          |4.9
-|RHOL 5.1|X            |X            |
-|RHOL 5.2|X            |X            |X
-|RHOL 5.3|             |X             |X
-|====
-
 include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
+[id="cluster-logging-ocp-compatibility"]
+= {product-title} compatibility
```

Comment:
```suggestion
== {product-title} compatibility
```

Reply:
![image](https://user-images.githubusercontent.com/350907/168106918-9b0025e2-cdc5-4288-a04b-2986c737e69e.png)
After you make this update, all these H2 headings should be at the same level.

Reply:
If there's a bug that prevents these H2s from rendering at the same level, try experimenting with changing line 9 to `include::modules/making-open-source-more-inclusive.adoc[leveloffset=+2]`